go/libkb/secret_store_other.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build !darwin,!android
+
+package libkb
+
+import "os"
+
+func NewSecretStoreAll(g *GlobalContext) SecretStoreAll {
+	// In order to not break production build releases, only
+	// use the SecretStoreFile on windows and linux if this
+	// environment variable is set.
```

Comment:
Is the idea here that we'll just use this var for our own testing, until we're ready for this to be in prod?